src/main/scala/com/ing/wbaa/rokku/proxy/provider/atlas/LineageHelpers.scala

```diff
@@ -143,33 +189,30 @@ trait LineageHelpers extends EventProducer {
     val userName = userSTS.userName.value
     val clientHost = clientIPAddress.getAddress().get().getHostAddress
     val clientType = lh.clientType.getOrElse("generic")
-    val bucketObject = lh.bucketObject.getOrElse("emptyObject")
+    val bucketObject = lh.bucketObject
     val pseudoDir = lh.pseduoDir.getOrElse(s"${lh.bucket}/")
-    val objectNameFromCopySrc = lh.copySource.getOrElse("")
+    val objectNameFromCopySrc = getObjectName(lh.copySource.getOrElse(""))
     val bucketNameFromCopySrc = lh.copySource.getOrElse("").split("/").head
```

Comment:
aws sdk creates  x-amz-copy-source: test/file3.txt from cmd and via sdk it actually creates it with leading slash, that leads to parsing error and empty bucket via sdk

Reply:
done